src/App.jsx

```diff
@@ -1,3 +1,14 @@
+import Header from './components/Layout/Header';
+import { ListProvider } from './context/ListContext';
+import ListView from './views/List/ListView';
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <>
+      <ListProvider>
```

Comment:
You will want to put your `<ListProvider/>` in your `index.jsx.` Putting it at the high level will ensure your entire app has access to it, shouldn't be an issue now since this is your only context and it's cascading to it's children.